### ISSUE TITLE
Gutter change indicators missing

### ISSUE THREAD

**Issue by nsgundy** [mrcrowl/vscode-hg#31]
VSCode: 1.17.2
Hg: 4.0
vscode-hg: 1.2.0

Issue:
The Hg change indicators are missing from the gutter. Not quite sure when this started happening, but it wasn't that long ago. The Git scm provider still shows them (on a different workspace). Everything else seems to be working. The badge on the left counts the changed files, the diffs show as normal and the status bar shows the current branch and the upstream push/pull count.
The output for hg looks pretty normal:

```
Using hg 4.0 from C:\Program Files\TortoiseHg\hg.exe
hg root
hg paths
hg summary -q
hg root
abort: no repository found in 'c:\Users\default\AppData\Roaming\Code\User' (.hg not found)!

hg branch
hg status -C
hg summary -q
hg branch
hg status -C
```

Any pointers as to what might have gone wrong?

**Comment by mrcrowl** [comment]
Apologies, this bug was introduced in 1.2.0.

A colleague pointed this out to me yesterday also. 

I hope to have it resolved soon.

**Comment by mrcrowl** [comment]
In the meantime, you could manually install an older version from the releases page using a vsix file.

**Comment by nsgundy** [comment]
Thanks for the tip. I had a bit of trouble at first installing the vsix file (first time I am trying that) as I kept getting version 1.2. All good though after I disabled auto-updating for extensions. The gutters are back with 1.1.6. Looking forward to the new version!

**Comment by nsgundy** [comment]
Confirmed fixed in `1.2.1`. Thanks!
